chalk-solve/src/clauses/builtin_traits.rs

```diff
@@ -30,3 +30,19 @@ pub fn add_builtin_program_clauses<I: Interner>(
         WellKnownTrait::DropTrait => (),
     }
 }
+
+/// Check the same trait for a list of types.
+pub fn needs_impl_for_tys<I: Interner>(
```

Comment:
I have mixed feelings about this helper function. While in theory it sounds helpful, there's a couple problems:
1) It's not even in the same file as its only *2* uses.
2) Both uses only use one clause.

Instead, I would just move this to `builtin_traits/sized.rs` and make it take just a single `Ty`

Reply:
It was requested by @Areredify because it's going to be used later (once this is merged) for clone and copy traits.

Reply:
Ah, fair enough.